__init__.py

```diff
@@ -172,26 +160,39 @@ def _handle_client_exception(self, callback, *args, **kwargs):
                     'url': exception.request.url})
         return False
 
+    # Intent handlers
     def handle_turn_on_intent(self, message):
         LOGGER.debug("Turn on intent on entity: "+message.data.get("entity"))
         message.data["Entity"] = message.data.get("entity")
         message.data["Action"] = "on"
-        self.handle_switch_intent(message)
+        self.handle_switch(message)
 
     def handle_turn_off_intent(self, message):
         LOGGER.debug(message.data)
         LOGGER.debug("Turn off intent on entity: "+message.data.get("entity"))
         message.data["Entity"] = message.data.get("entity")
         message.data["Action"] = "off"
-        self.handle_switch_intent(message)
+        self.handle_switch(message)
 
     def handle_toggle_intent(self, message):
         LOGGER.debug("Toggle intent on entity: " + message.data.get("entity"))
         message.data["Entity"] = message.data.get("entity")
         message.data["Action"] = "toggle"
-        self.handle_switch_intent(message)
+        self.handle_switch(message)
+
+    def handle_sensor_intent(self, message):
+        LOGGER.debug("Turn on intent on entity: "+message.data.get("entity"))
+        message.data["Entity"] = message.data.get("entity")
+        self.handle_sensor(message)
 
-    def handle_switch_intent(self, message):
+    def handle_light_set_intent(self, message):
+        LOGGER.debug("Change light intensity: "+message.data.get("entity") \
+            +"to"+message.data.get("brightnessvalue")+"percent")
+        message.data["Entity"] = message.data.get("entity")
+        message.data["Brightnessvalue"] = message.data.get("brightnessvalue")
+        self.handle_light_set(message)
+
+    def handle_switch(self, message):
```

Comment:
With intent hook removed, should we make this private ( _handle_switch)?

Reply:
can be - will change

Reply:
done